packages/edit-site/src/components/sync-state-with-url/use-init-edited-entity-from-url.js

```diff
@@ -63,10 +63,10 @@ function useResolveEditedEntityAndContext( { path, postId, postType } ) {
 				: false;
 		}
 		return {
-			hasLoadedAllDependencies: !! base && !! siteData,
+			hasLoadedAllDependencies: !! siteData,
 			homepageId: _homepageId,
 			postsPageId: _postsPageId,
-			url: base?.home,
+			url: siteData?.home,
```

Comment:
We're in the site editor, [which is only accessible to admins](https://github.com/WordPress/gutenberg/pull/61088#discussion_r1579207667), so we can use the normal settings.